release/models/system/openconfig-system-logging.yang

```diff
@@ -426,6 +432,12 @@ module openconfig-system-logging {
         "Source IP address for packets to the log server";
     }
 
+    leaf network-instance {
```

Comment:
Reminder that the lone addition of the `network-instance` will not allow duplicate IP addresses on different network instances even if it's a valid configuration. This is the same situation as being discussed in #767